superset/assets/visualizations/time_table.jsx

```diff
@@ -53,13 +54,12 @@ function viz(slice, payload) {
   }
   const tableData = metrics.map((metric) => {
     let leftCell;
-    const context = Object.assign({}, fd, { metric });
-    const url = fd.url ? Mustache.render(fd.url, context) : null;
```

Comment:
Why was Mustache removed? The url needs to be rendered with the context (to interpret something like {{ metric }}).

Reply:
ohhh, yeah I didn't understand what this was doing fully/thought it was forcing the url to be within the app. added it back.